make-release.sh

```diff
@@ -6,24 +6,45 @@
 # set to 1 to actually trigger changes in the release branch
 TRIGGER_RELEASE=0 
 NOCOMMIT=0
+USE_TMP_DIR=0
+REPO=git@github.com:eclipse/che-devfile-registry
 
 while [[ "$#" -gt 0 ]]; do
   case $1 in
     '-t'|'--trigger-release') TRIGGER_RELEASE=1; NOCOMMIT=0; shift 0;;
-    '-r'|'--repo') REPO="$2"; shift 1;;
     '-v'|'--version') VERSION="$2"; shift 1;;
+    '-tmp'|'--use-tmp-dir')USE_TMP_DIR=1; shift 0;;
```

Comment:
if you wanted to save a variable you could use `--use-tmp-dir` flag to set `TMP` directly with `$(mktemp -d)` here... and then later either you check ``[[ $TMP != "" ]] or that `[[ -d $TMP ]]`.

Not  a dealbreaker, just a way to remove one exta variable from the script as we don't need a boolean AND a directory.